content/millennium/r4/summary/family-member-history.md

```diff
@@ -0,0 +1,198 @@
+---
+title: FamilyMemberHistory | R4 API
+---
+
+# FamilyMemberHistory
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The FamilyMemberHistory resource represents family history for a given patient. It may provide a list of conditions associated to a patient's family member or the absence of a condition on a given individual. In other cases, the resource may indicate that there is no relevant family history, no significant history for a given relative, or that the patient’s family history is unknown or unable to obtain.
+
+The resource should be leveraged as a snapshot in time and new data should be consistently retrieved through the API rather than stored within an application. There may be instances where a relative had an associated condition removed from the patient’s chart which would then no longer be exposed on a subsequent transaction. It is crucial that the data be refreshed with every interaction.
+
+The following fields are returned if valued:
+
+* [FamilyMemberHistory id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+ 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for FamilyMemberHistory that meet supplied query parameters:
+
+    GET /FamilyMemberHistory?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Parameters
+
+Name      | Required?       | Type          | Description
+----------|-----------------|---------------|-------------------------------------------------------
+`_id`     | This or patient | [`token`]     | The logical resource id associated with the resource.
+`patient` | This or _id     | [`reference`] | The identity of a subject to list family member history items for. Example: `12345`
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/FamilyMemberHistory?_id=123-456
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_family_member_history_bundle) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+<%= beta_tag(action: true) %>
+
+List an individual FamilyMemberHistory by its id:
+
+    GET /FamilyMemberHistory/:id
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/FamilyMemberHistory/123-456
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_family_member_history_entry) %>
+
+<%= disclaimer %>
+
+## Create
+
+<%= beta_tag(action: true) %>
+
+Create new family member history.
```

Comment:
Should this maybe be 'Create a new FamilyMemberHistory.'

I've seen other two word resources (FinancialTransaction, RelatedPerson, etc) do it that way, but I've also seen some like this is currently - so up to you. It just sticks out to me that Create is like this but Update and others refer to it as FamilyMemberHistory

Reply:
I like it your way https://github.com/cerner/fhir.cerner.com/pull/660/commits/8b441b3fc17a938a03c796812854104db997ead0